### ISSUE TITLE
New Edition-Node installation does not have package.json patternlab commands copied

### ISSUE THREAD

**Issue by bmuenzenmeyer** [pattern-lab/patternlab-node#863]
Following the new installation instructions:

1. 
    ```bash
    mkdir new-project
    cd new-project
    npm init -y && npx -p @pattern-lab/cli -c 'patternlab init'
    ```
1. Follow the on-screen prompts to choose your Edition and a Starterkit should you want one.
  * `Current directory`
  * `edition-node`

Expected behavior: Installation of `Edition-Node` into the current working directory should add [the default commands](https://github.com/pattern-lab/patternlab-node/blob/master/packages/edition-node/package.json#L22-L26) into the current directory's package.json file
Actual behavior: does not do this

**Comment by raphaelokon** [comment]
My gut feeling is that we shouldn't tamper with the package.json → A more soft idea would be to display a message to copy those commands in there.

**Comment by raphaelokon** [comment]
I've implemented it nevertheless … once #864 is merged I can push it to its branch and we can decide
